mne/io/brainvision/brainvision.py

```diff
@@ -70,7 +71,7 @@ class RawBrainVision(_BaseRaw):
     """
     @verbose
     def __init__(self, vhdr_fname, montage=None,
-                 eog=('HEOGL', 'HEOGR', 'VEOGb'), misc=(),
+                 eog=('HEOGL', 'HEOGR', 'VEOGb'), misc='auto',
```

Comment:
this is an API change so it needs to be noted in `whats_new` that now channels with non-`uV` units will be turned to `misc` (if that is indeed what happens)